NAMESPACE

```diff
@@ -21,6 +21,9 @@ export(getConceptSetName)
 export(getConceptSetsAndConceptsFromCohort)
 export(getConcepts)
 export(getEstimation)
+export(getIncidenceRateDefinitions)
```

Comment:
I like having these 3 functions side-by-side. Maybe we should make this a pattern we apply consistently everywhere?

Reply:
Nice - the pattern being

- get definition (specification, expression as R object)
- get generation information (has it run?)
- get output (or results)

correct?